packages/frontend/pages/MessageBoard.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useRef, useEffect} from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export function Message( {message} ) {
+    return (
+        <div>
+            <h4>Name: {message.name}</h4>
+            <p>Message:</p>
+            <p>{message.msg}</p>
+            <p>[DEBUG]MessageID: {message.id}</p>
```

Comment:
Can be removed

Reply:
moved whole function to own file (Message.tsx), i hope this is what you meant, else please elaborate